test/acceptance/configmap_test.go

```diff
@@ -29,7 +29,7 @@ func TestKubernetesManifest_ConfigMap(t *testing.T) {
 		"namespace": namespace,
 		"name":      name,
 	}
-	tfconfig := loadTerraformConfig(t, "configmap.tf", tfvars)
+	tfconfig := loadTerraformConfig(t, "ConfigMap/configmap.tf", tfvars)
 	tf.RequireSetConfig(t, tfconfig)
```

Comment:
These tests look really neat! I like how readable they are. There's only one part I'm having trouble with. My IDE is unable to tell me anything about `tf` in this context:

```
tf := tfhelper.RequireNewWorkingDir(t)
```

So, for example, when I was going to check and see if `tf.RequireValidate()` might exist, it wasn't able to pull in any information about tf. Are you seeing this issue on your IDE too?

I think it's happening because `tfhelper` is not imported in this file, or passed into the function. Instead, it seems to be referenced via a global variable:

https://github.com/hashicorp/terraform-provider-kubernetes-alpha/blob/47fa902c75267087513e4f79ed4d92084b16ea94/test/acceptance/acceptance_test.go#L23

But, if I import the library into the file, and create a local variable instead, it solves the problem:

```
import (
...	
    tftest "github.com/hashicorp/terraform-plugin-test/v2"
...
)

func TestKubernetesManifest_ConfigMap(t *testing.T) {
	name := randName()
	namespace := randName()

	var tfhelper *tftest.Helper
...
```

What do you think about this pattern? It would enable contributors to look up information about the test helper using their IDE, and it would help IDEs to be able to show error messages about the usage of this library.

Reply:
You're right. I had the same issues in my IDE and did notice the global variables but was trying to keep changes focused on just enabling the tests to run stable. I do like the suggestion you posted so I'll have a go at including it. It should not be difficult to make those changes and remove the global.

Reply:
I was having a look at the import / variable visibility issue and it looks like some of the New...() functions of the various helpers are pretty heavy - they create the go-client instance, for example. Repeating that for every test seems a bit intensive.
I think we need a way to get rid of the package level variables without repeating the construction of those helpers.

I would prefer to push that bit of work to a future PR as this has already been eating a lot more than planned for, given the terraform binary issues and debugging. Are you OK with that approach? Feel free to experiment with this and propose changes or otherwise let's plan this work in a future sprint.